lib/gameMenu.js

```diff
@@ -3,11 +3,15 @@ require('./github_link')
 require('./components/button')
 require('./components/buttonlink')
 require('./components/textfield')
+require('./optionsMenu')
+const optionsMenu = document.getElementById('options-menu')
 
 class GameMenu extends LitElement {
   constructor () {
     super()
     this.inMenu = false
+    if (optionsMenu === null) console.log('Options menu is null... idk why')
```

Comment:
does this actually happen? can you console.trace it?

Reply:
That’s just for debugging—I messed up my code before and couldn’t figure out what happened; I’ll remove it before merging.

Reply:
Removed this line; will push soon